src/resources/lang/enso/ro.json

```diff
@@ -1,6 +1,151 @@
 {
-    "Key": null,
+    "Die Aktion war erfolgreich": null,
+    "App": null,
     "Value": null,
+    "Key": null,
+    "Core": null,
+    "Only missing": null,
+    "edit texts": null,
+    "keys found": null,
+    "German": null,
+    "Choose language": null,
+    "Edit Tutorial": null,
+    "Error": null,
+    "Postal Area": null,
+    "Localisation Index": null,
+    "Time": null,
+    "Imported Entries": null,
+    "Import Summary": null,
+    "File": null,
+    "Line": null,
+    "Delete Template": null,
+    "Download Template": null,
+    "Import": null,
+    "Upload Template": null,
+    "Imported At": null,
+    "Imported By": null,
+    "File name": null,
+    "Successful": null,
+    "Issues": null,
+    "ImportType": null,
+    "Past Imports": null,
+    "data import": null,
+    "The tutorial was created!": null,
+    "Permission": null,
+    "Tutorials Index": null,
+    "The log was cleaned": null,
+    "The log file": null,
+    "Current file size is": null,
+    "was last updated": null,
+    "Show Log": null,
+    "MB": null,
```

Comment:
?

Reply:
not from me and not german - I think it's megabyte?

Reply:
I think you're right, somewhere in the logs index / show